x/distribution/types/validator.go

```diff
@@ -5,9 +5,25 @@ import (
 )
 
 // historical rewards for a validator
-// TODO add reference counter, ref https://github.com/cosmos/cosmos-sdk/pull/3099#discussion_r245747051
 // height is implicit within the store key
-type ValidatorHistoricalRewards = sdk.DecCoins
+// the reference count indicates the number of objects
+// which might need to reference this historical entry
+// at any point, it is equal to the number of outstanding
+// delegations starting on the associated period, plus
+// the number of slashes at the associated period, plus one
+// per validator for the zeroeth period
+type ValidatorHistoricalRewards struct {
+	SeqValue       sdk.DecCoins `json:"seq_value"`
```

Comment:
Can we think of a slightly better field for sequence value? This refers to the rewards, correct?

Reply:
`PeriodRewards`?

Reply:
Renamed - although note it's *not* the rewards for the period, it's the sum from the first period of all rewards divided by validator tokens.

Reply:
I see, I see, so it's a ratio.

Reply:
Yeah I think so (sorry @cwgoes) -- maybe `RewardRatio` or some variant?

Reply:
Renamed to `CumulativeRewardRatio` - clear?